Updated gitignore and chain files

effofxprime opened the pull request:
There were a few odd issues with some of the RPCs blocking certain calls for some chains.  I don't know all affected, as I only use a few that the true ping.pub repository uses. Stargaze was one.  Lum was no longer working and I added a new endpoint that was given to me by someone in their Discord handle:  @Wetty#0028 if they need to be contacted.

I added a new chain, Beezee.  They are new and are working on getting an Osmosis pool going but have been running mainnet for a couple of months since their swap ended.
website: https://getbze.com/
Discord:  https://discord.gg/yd4SGwRTHP


I have added Polkachu and Notional API and RPC endpoints.  I put them in front, and that may be an issue for some chains that want their main API listed first.  This was an afterthought and if it needs to be corrected, let me know and I will fix it right up.
@Polkachu @notional-labs

I have updated the git ignore file to exclude some of the yarn files created when running that command.

PolkachuIntern commented:
Most API endpoints might not work because of CORS

liangping commented:
please check those endpoints are enabled CORS

effofxprime commented:
Regarding the Notional API endpoints, I contacted Notional and asked if I could use them and they said it was no problem.  Let me verify if they would also need to allow CORS for you as well Ping.  Im unsure if they wildcard their CORS or whitetlist it.

For Polka's endpoints, the ones that I have put out in front (Like stargaze), they are currently working for me without issue and without CORS blocking from what I am seeing in the console.

I have experience with github on a personal level but not much when it comes to pushing PR's to a master repo.  I wanted to make the commit 2d2733a a separate merge request but it has merged itself with this commit.  How can I separate them?

I have tested the changed endpoints with the corresponding chains and they all are working, and I detected no massive CORS blocking that isn't already known.  IE: LUM network is still in an odd place where calls for blocks are getting 501'd and we're unsure why calls to that cannot be made.

effofxprime commented:
Live version of the above changes is running here: https://explorer.erialos.me/

effofxprime commented:
Regarding LUM and possibly a few other networks.

Lum core team reported to me that parts of REST have been depreciated for the Cosmos SDK version they are currently on.  `v0.45.7`
I am unsure about this myself.  The rest endpoint does not respond like most other rest endpoints with an interactive front end to query the chain.  I know REST was supposed to get depreciated in the future and replaced with gRPC, no idea what versions though.

liangping commented on src/chains/mainnet/osmosis.json:
```diff
@@ -21,11 +21,5 @@
         "exponent": "6",
         "coingecko_id": "ion", 
         "logo": "/logos/osmosis.jpg"
-    },{
-        "base": "usomm",
-        "symbol": "SOMM",
-        "exponent": "6",
-        "coingecko_id": "somm", 
-        "logo": ""
```
we need keep SOMM here for now.  can you add it back

liangping commented on src/chains/mainnet/secret.json:
```diff
@@ -1,10 +1,10 @@
+    "rpc": ["http://beta-api.scrt.network:26657", "https://api.scrt.network:443"],
     "snapshot_provider": "",
-    "sdk_version": "0.45.4",
+    "sdk_version": "0.33.8",
```
this version is probably too old.

liangping commented on src/chains/mainnet/sifchain.json:
```diff
@@ -1,9 +1,9 @@
+    "rpc": ["https://rpc-sifchain-ia.cosmosia.notional.ventures/", "https://sifchain-rpc.polkachu.com:443", "https://rpc.sifchain.finance:443", "https://rpc-archive.sifchain.finance:443"],
     "snapshot_provider": "",
-    "sdk_version": "0.45.0",
+    "sdk_version": "0.42.6",
```
please check the version

effofxprime commented:
This morning, I went through and manually checked most of the chain files to their github SDK version and updated them.

I corrected the files you spotted a bad change on.

Added Beezee and Aura logo to `/public/logos`

*note*
Some chains had the wrong repo link, Decenttr was one.  Others had no Gitthub link, Echelon, Stargaze, Stafihub, were a few to mention.

effofxprime commented:
Lum network is mostly resolved.  However, no blocks or transactions are showing.  I spoke with one of the core team members and I was instructed to use RPC over REST, which doesn't really resolve the issue.  I've noticed a few other chains that have a similar issue where their REST api's do not serve everything that most others do.

Maybe worth looking into adding gRPC support down the road?

liangping commented:
@effofxprime Please fix
- Aura endpoint don't enable CORS. 
- TERITORI has no logo.
